regression-test/src/app/components/MODAL_IUI.js

```diff
@@ -26,6 +26,7 @@ import { Modal } from '@instructure/ui'
 import React from 'react'
 const MODAL_IUI = () => {
   return (
+    // <></>
```

Comment:
Maybe this was just left here by accident.

Reply:
I removed it, thanks.